docs/token-api.md

```diff
@@ -34,69 +35,84 @@ parties, called `redeemers`, can invoke this operation.
+    - `Precision`: The precision used to represent the token quantity.
+    - `MaxTokenValue`: It is the maximum quantity that a token can contain.
+    - `TokenDataHiding`: When true it means that the content of the tokens is hidden.
+    - `GraphHiding`:  When true it means that the tokens are untraceable.
+    - `Auditors`: A list of auditor identities.
+
+The developer can access the `Public Parameters Manager` of a give TMS as follows:
+```go
+    ppm := tms.PublicParametersManager()
+```
+
+## Wallet Manager
+
+A Wallet consists of a long-term identity and all its derivation (if any).
+Examples of long-term identities are:
     - An `X509 Certificate` for an ECDSA signing public-key
     - An `Idemix Credential`. In this case, the wallet will contain also all pseudonyms derived from the credential.
   
-  However, it is always the specific driver that dictates what a long-term identity is.
+However, it is always the specific driver that dictates what a long-term identity is.
 
-  All operations that require a signature refer to wallets to identify the signing and verification keys.
-  There are wallets for `Token Issuers`, `Owners`, and `Auditors`.
-  Depending on the nature of the wallet additional information can be extracted like:
+All operations that require a signature refer to wallets to identify the signing and verification keys.
+There are wallets for `Issuers`, `Owners`, and `Auditors`.
+Depending on the nature of the wallet additional information can be extracted like:
     - An Issuer Wallet gives access to the list of issued tokens;
-    - A Token Owner Wallet gives access to the list of owned unspent tokens;
-
-  The Wallet Manager offers API to manage wallets.
-- `Token Request`: The Token Request is a container of token actions (issue, transfer, and redeem) that must be
-  performed atomically.
-  The Token Request offers API to add actions or inspect actions already present in the container.
-  More information in Section [Token Request](#token-request).
-
-- `Validator`: The validator is the component that sets the validation rules for a Token Request. The rules depend on the
-  type of tokens supported (fungible, non-fungible, and so on), and on the specific driver implementation.
-  A Validator validates Token Requests with the respect to:
-    - A given Anchor (e.g., Fabric TxID), and
-    - The Ledger. Notice that, in certain implementations, the ledger might not be needed.
-
-  Even though, certain validation rules are driver specific, we can identify the following general validation rules.
-  A token request should:
-    - Well-formed, and
-    - Satisfies the constraints of the payment system. Namely:
-        - Only the `rightful owner` can transfer a token,
-        - No token can be created out of the blue,
-        - Audited(able)
-        - Etc. (Each implementation can enforce additional requirements, if needed)
-  
+    - An Owner Wallet gives access to the list of owned unspent tokens;
```

Comment:
mh ... is that true? Technically we can get the tokens from the Token Vault using the wallet ID as filter?!

Reply:
True, but also the Owner Wallet interface allows that.